invoker.gemspec

```diff
@@ -36,6 +36,7 @@ Gem::Specification.new do |s|
   s.add_dependency("facter", "~> 2.2")
   s.add_dependency("http-parser-lite", "~> 0.6")
   s.add_dependency("dotenv", "~> 1.0.2")
+  s.add_dependency("libnotify", "~> 0.9.1")
```

Comment:
Is this gem installable in OSX btw? That is one of the reasons, I don't want this gem to be explicit dependency.